pl_examples/basic_examples/README.md

```diff
@@ -2,62 +2,70 @@
 
 Use these examples to test how Lightning works.
 
-#### MNIST
+## MNIST Examples
 
-Trains MNIST where the model is defined inside the `LightningModule`.
+Here are 5 MNIST examples showing you how to gradually convert from pure PyTorch to PyTorch Lightning.
+
+The transition through [LightningLite](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.rst) from pure PyTorch is optional but it might be helpful to learn about it.
+
+#### 1. Image Classifier with Vanilla PyTorch
+
+Trains a simple CNN over MNIST using vanilla PyTorch.
 
 ```bash
 # cpu
-python simple_image_classifier.py
+python image_classifier_1_pytorch.py
```

Comment:
what is the number? could be called it case/chapter?
```suggestion
python image_classifier_case-1_pytorch.py
```

Reply:
the number just tells the reader in which order they need to look at the examples. 
The files get sorted normally, so it should be pretty straightforward

Reply:
yes I know what it is for, but it is a bit confusing without context... but it is minor comment